test/lib/Resource.js

```diff
@@ -160,6 +185,12 @@ test("Resource: clone resource with stream", (t) => {
 	});
 });
 
+test("getStream for empty file: correctly retrieved", async (t) => {
```

Comment:
This test is unrelated to your fix. It also passes without it. Why is it necessary? The commit message does not explain that.

Reply:
added it for completeness, there was no test for a stream of an empty file. Will add it to the commit message.

Reply:
will update message

Reply:
Well I'm just wondering because half of the changes done in this PR are related to this test. And I'm not sure whether it tests any of our code.

In this test case, you pass in a stream (using the `createStream` callback) and retrieve it using `getStream`. The Resource implementation never handles the streams content. So it doesn't matter whether it's empty or not. Maybe I'm missing something but to me this test seems pretty useless?

Reply:
removed it